.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,139 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
Where have you generated this config from?

Reply:
this was generated by AS after applying the Kotlin style to the project and deleting java.util.* from the wildcard imports and setting the wildcard import point to 99 entries (we've had problems with that being flagged by Detekt)